Hitting GH rate limits during tests

**Describe the bug**
```
/home/runner/work/mondoo-operator/mondoo-operator/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
go generate ./controllers/... ./pkg/... ./tests/framework/nexus/...
test -s /home/runner/work/mondoo-operator/mondoo-operator/bin/kustomize || { curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | bash -s -- 4.5.7 /home/runner/work/mondoo-operator/mondoo-operator/bin; }
Github rate-limiter failed the request. Either authenticate or wait a couple of minutes.
make: *** [Makefile:256: /home/runner/work/mondoo-operator/mondoo-operator/bin/kustomize] Error 1
```

**To Reproduce**
Looks like this does not happen every time.

**Expected behavior**
kustomize should install and tests start

--- a/Makefile
+++ b/Makefile
@@ -250,10 +250,15 @@ KUSTOMIZE_VERSION ?= v4.5.7
 CONTROLLER_TOOLS_VERSION ?= v0.10.0
 
 KUSTOMIZE_INSTALL_SCRIPT ?= "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
+## we intentionally install kustomize without the script to prevent GitHub API rate limits
 .PHONY: kustomize
 kustomize: $(KUSTOMIZE) ## Download kustomize locally if necessary.
 $(KUSTOMIZE): $(LOCALBIN)
-	test -s $(LOCALBIN)/kustomize || { curl -s $(KUSTOMIZE_INSTALL_SCRIPT) | bash -s -- $(subst v,,$(KUSTOMIZE_VERSION)) $(LOCALBIN); }
+ifeq (,$(wildcard $(LOCALBIN)/kustomize))
+	curl -sLO https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2F$(KUSTOMIZE_VERSION)/kustomize_$(KUSTOMIZE_VERSION)_linux_amd64.tar.gz && \
+	tar xzf kustomize_$(KUSTOMIZE_VERSION)_linux_amd64.tar.gz -C $(LOCALBIN)/ && \
+	rm kustomize_$(KUSTOMIZE_VERSION)_linux_amd64.tar.gz
+endif
 
 .PHONY: controller-gen
 controller-gen: $(CONTROLLER_GEN) ## Download controller-gen locally if necessary.

